src/lib/eliom_client.client.ml

```diff
@@ -402,15 +402,9 @@ let call_ocaml_service
       ?keep_nl_params ?nl_params ?keep_get_na_params
       ?progress ?upload_progress ?override_mime_type
       get_params post_params in
-  let%lwt () = Lwt_mutex.lock load_mutex in
```

Comment:
Removing this looks suspicious. @vouillon added the lock in 123bfc49 to "prevent an OCaml service to be called while loading a page".